setting/schema.json

```diff
@@ -395,6 +396,17 @@
                     ],
                     "type": "string"
                 },
+                "conventions": {
+                    "default": "None",
+                    "description": "* global-element\n* ...",
+                    "enum": [
```

Comment:
What is the meaning of these options?
Is there a documentation for our changes?

Reply:
In general, the changes in this file tell VSCode that these settings exist and are valid for the settings.xml
The meaning of the different settings and their possible values are documented here: https://github.com/LuaLS/lua-language-server/wiki/Settings#diagnosticsgroupfilestatus

But maybe in short: "fileStatus" set to none means that the diagnostic (group) is disabled unless the setting is explicitly changed. The description for the group is the tooltip in VSCode, and here generally just lists the diagnostics that are contained in a certain group